tests/src/JIT/CodeGenBringUpTests/LocallocCnstB32_Loop.cs

```diff
@@ -0,0 +1,51 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+
+
+using System;
+using System.Runtime.CompilerServices;
+public class BringUpTest
+{
+    const int Pass = 100;
+    const int Fail = -1;
+
+    // Reduce all values to byte
+    [MethodImplAttribute(MethodImplOptions.NoInlining)]
+    public static unsafe bool CHECK(byte check, byte expected) {
+        return check == expected;
+    }
+
+    [MethodImplAttribute(MethodImplOptions.NoInlining)]
+    public static unsafe int LocallocCnstB32_Loop(int count)
+    {
+        for (int j = 0; j < count; j++)
+        {
+            byte* a = stackalloc byte[32];
+            for (int i = 0; i < 5; i++)
+            {
+                a[i] = (byte) (i + j);
+            }
+
+            for (int i = 0; i < 5; i++)
+            {
+                if (!CHECK(a[i], (byte) (i + j))) return i + j * 100;
```

Comment:
Why is this written as a loop with a return statement?
It is unclear how many time this loop iterates.

Reply:
I just figured out that any early return is a failure.

Reply:
fyi, I introduced the loop because otherwise the JIT converts a constant size localloc to a local var.